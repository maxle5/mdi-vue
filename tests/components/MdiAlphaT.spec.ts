
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiAlphaT from "../../src/components/MdiAlphaT.vue";

test("MdiAlphaT snapshot", () => {
  const wrapper = mount(MdiAlphaT, {});
  expect(wrapper.html()).toMatchSnapshot();
});
