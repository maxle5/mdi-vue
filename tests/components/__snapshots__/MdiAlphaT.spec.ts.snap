// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiAlphaT snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M9,7V9H11V17H13V9H15V7H9Z" style="fill: currentColor;"></path>
</svg>"
`;
